developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md

```diff
@@ -1,244 +1,251 @@
+                for (int count = 1; count < 31; ++count)
+                {
+                    Thread.Sleep(1000);
+                    Console.WriteLine(count + " seconds elapsed");
+                }
+
+                Console.WriteLine();
+                Console.WriteLine("Job run results.");
+                Console.WriteLine();
+
+                // Since the PowerShell job runs in a non-interactive Task Scheduler
+                // process the job status and output data is written to a file based
+                // job store and the directory location is the current user local app
+                // data ($env:LOCALAPPDATA).
+                // This job store can be accessed through the ScheduledJobSourceAdapter class.
+                ScheduledJobSourceAdapter schedJobSourceAdpater = new ScheduledJobSourceAdapter();
+                IList<Job2> jobRuns = schedJobSourceAdpater.GetJobs();
+                foreach (var jobRun in jobRuns)
+                {
+                    // Check for jobs in finished state.
+                    // Note that job data is not written to the job store until the job
+                    // has reached a finished state.
+                    JobState jobState = jobRun.JobStateInfo.State;
+                    if (jobState == JobState.Completed ||
+                        jobState == JobState.Stopped ||
+                        jobState == JobState.Failed)
+                    {
+                        // Write job run finished state.
+                        Console.WriteLine("Job Status: " + jobRun.JobStateInfo.State);
+                        Console.WriteLine();
+                        Console.WriteLine("Job Data");
+
+                        // Write output data.
+                        foreach (var item in jobRun.Output)
+                        {
+                            Console.WriteLine(item.ToString());
+                        }
+
+                        // Write any errors.
+                        foreach (var item in jobRun.Error)
+                        {
+                            Console.WriteLine(item.ToString());
+                        }
+                    }
+                }
+
+                Console.WriteLine();
+                Console.WriteLine("Press any key to continue...");
+                Console.ReadKey();
+            }
+            catch (ScheduledJobException e)
+            {
+                Console.WriteLine("Error: " + e.Message);
+            }
+            finally
+            {
+                // Unregister this scheduled job or an error will be thrown when
+                // running this sample code again (scheduled job already exists.)
+                // because all registered scheduled jobs must have a unique name.
+                if (registrationSucceeded)
+                {
+                    schedJobDefinition.Remove(true);
+                }
+            }
+        }
+    }
+}
+
+```


```

Comment:
Control character

Reply:
This was an issue with my code fence logic and is fixed